runtime/oti/vm_api.h

```diff
@@ -357,6 +357,18 @@ peekClassHashTable(J9VMThread* currentThread, J9ClassLoader* classLoader, U_8* c
 J9Class*
 internalCreateArrayClass(J9VMThread* vmThread, J9ROMArrayClass* romClass, J9Class* elementClass);
 
+/**
+* @brief
+* @param vmThread
+* @param romClass
+* @param elementClass
+* @param options
+* @return J9Class*
+*/
+J9Class*
+internalCreateArrayClassWithOptions(J9VMThread* vmThread, J9ROMArrayClass* romClass, J9Class* elementClass, UDATA options);
+
```

Comment:
Please complete this documentation.
The asterisk of the second and subsequent lines of a multi-line comment should line up with the one in the first line.
Space before, not after `*`.
Blank line 370 is not needed.